pkg/apis/acid.zalan.do/v1/operator_configuration_type.go

```diff
@@ -185,6 +199,7 @@ type OperatorConfigurationData struct {
 	LoggingRESTAPI             LoggingRESTAPIConfiguration        `json:"logging_rest_api"`
 	Scalyr                     ScalyrConfiguration                `json:"scalyr"`
 	LogicalBackup              OperatorLogicalBackupConfiguration `json:"logical_backup"`
+	ConnectionPool             ConnectionPoolConfiguration        `json:"connection_pool"`
```

Comment:
What is the idea here?

Reply:
What's an idea behind operator default values for pool configs, or what?